src/utils/utils.ts

```diff
@@ -133,10 +134,14 @@ export const apiCall = async (apiUrl: string) => {
 
 export const getItemDataFromImageID = async (imageID: string) => {
   const apiUri = await getAPIUri("local_image_id", imageID);
-  const data = await apiCall(apiUri.apiUri);
+  // const data = await apiCall(apiUri.apiUri);
```

Comment:
This still makes a call to `getAPIUri`, but I think it's minor considering that most API calls have been replaced.